k8s/preview/common-overlay/hmc/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
u need a similar change for AAT, this PR would have broken flux on AAT.

Reply:
thanks please see https://github.com/hmcts/cnp-flux-config/pull/9746